finch/processes/wps_xsubset_point_dataset.py

```diff
@@ -94,11 +92,16 @@ def _handler(self, request: WPSRequest, response: ExecuteResponse):
 
         variable = request.inputs["variable"][0].data
         variables = None if variable is None else [variable]
-        rcp = single_input_or_none(request.inputs, "rcp")
+        scenario = single_input_or_none(request.inputs, "scenario")
```

Comment:
@aulemahal Would it make sense to have backward compatibility on `rcp` and `dataset_name` (use `rcp` and `dataset_name` if `scenario` or `dataset` isn't defined)? This way this PR would not represent a breaking change and related to renamed fields.

Reply:
That's a bit heavier than the `bccaqv2` compatibility. It would mean having the two arguments in each processes? "scenario" + "rcp"?  I'll look into this.

Also, `dataset_name` was broken, it simply didn't work, at least for the indicator ensemble processes. So I'm guessing it isn't a breaking change?

Reply:
Forget about the compatibility if that's more work (and anyway doesn't make sense that `rcp` translates to `ssp` values). 
Changes related to `dataset_name`/`dataset` will be made on the portal itself so no worries about this change.